pom.xml

```diff
@@ -340,6 +341,9 @@
         <plugin>
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>2.10.1</version>
+          <configuration>
+            <quiet>${javadoc.quiet}</quiet>
```

Comment:
Agreed, I see no particular use case for making it verbose, so it would be simply and better to just say

```xml
<quiet>true</quiet>
```

and if someone really wants the opposite, they can add a configuration fragment in their POM to override.